volatility/framework/plugins/mac/pslist_process_groups.py

```diff
@@ -0,0 +1,89 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
```

Comment:
I think we decided that when new files were added they need to have the right year in the header?  Existing files don't need to be updated though...  5:S

Reply:
included in the new commit.